.github/sast_configurations/codeql.yml

```diff
@@ -0,0 +1,9 @@
+name: "CodeQL configurations"
+
+query-filters:
+  - include:
+      tags contain: security
```

Comment:
Is this key name correct? Having a space in it seems dubious.

Reply:
Yes, this is correct. I have tested it to be sure.

Reply:
I'd recommend quoting it in that case. (someday soon we'll have actionlint, which should help with this sort of thing!)
```suggestion
      "tags contain": security
```